ding/bonus/ppof.py

```diff
@@ -16,6 +17,11 @@
 from .config import get_instance_config, get_instance_env, get_hybrid_shape
 
 
+@dataclass
+class TrainingReturn:
+    wandb_url: str
+
```

Comment:
Add a data class return for catching wandb run urls when doing task initialization.